Lib/test/test_capi/test_opt.py

```diff
@@ -1564,6 +1564,23 @@ def f(n):
         # But all of the appends we care about are still there:
         self.assertEqual(uops.count("_CALL_LIST_APPEND"), len("ABCDEFG"))
 
+def test_compare_pop_two_load_const_inline_borrow(self):
+        def testfunc(n):
+            x = 0
+            for _ in range(n):
+                a = 10
+                b = 10
+                if a == b:
+                    x += 1
+            return x
+
+        res, ex = self._run_with_optimizer(testfunc, TIER2_THRESHOLD)
+        self.assertEqual(res, 1)
```

Comment:
The return value should be `TIER2_THRESHOLD`, since that's the number of times the loop runs.

```suggestion
        self.assertEqual(res, TIER2_THRESHOLD)
```

This test isn't failing because it needs to be indented to be part of the enclosing class. ;)

Reply:
Ohhh, I was wondering why this passed. It felt off to me. Thanks!